app/models/core/Nino.scala

```diff
@@ -16,4 +16,10 @@
 
 package models.core
 
+import play.api.libs.json.{Format, Json}
+
 case class Nino(value: String) extends AnyVal
+
+object Nino {
```

Comment:
this might be a redundant as conversion is really required for complex types as far as I know;